can/interfaces/pcan/PCANBasic.py

```diff
@@ -287,36 +269,29 @@
 PCAN_TYPE_DNG_SJA        = TPCANType(0x05)  # PCAN-Dongle SJA1000
 PCAN_TYPE_DNG_SJA_EPP    = TPCANType(0x06)  # PCAN-Dongle EPP SJA1000
 
-# Represents a PCAN message
-#
 class TPCANMsg (Structure):
     """
     Represents a PCAN message
     """
-    _fields_ = [ ("ID",      c_uint),          # 11/29-bit message identifier
+    _fields_ = [ ("ID",      c_uint),           # 11/29-bit message identifier
                  ("MSGTYPE", TPCANMessageType), # Type of the message
                  ("LEN",     c_ubyte),          # Data Length Code of the message (0..8)
                  ("DATA",    c_ubyte * 8) ]     # Data of the message (DATA[0]..DATA[7])
 
-# Represents a timestamp of a received PCAN message
-# Total Microseconds = micros + 1000 * millis + 0x100000000 * 1000 * millis_overflow
```

Comment:
You've deleted a lot of comments from this file.

Reply:
Yes, I did. But the comments beginning with `#` above each method were simply the first line of the actual method doc string, so I removed the redundancy. They were simply a duplicate and distracting.